XLS-47d-PriceOracles/README.md

```diff
@@ -0,0 +1,453 @@
+- `SymbolClass` describes a type of the assets, for instance "currency", "commodity", "index". It is a string of up to ten ASCII hex encoded characters (0x20-0x7E).
+- `LastUpdateTime` is the specific point in time when the data was last updated. The `LastUpdateTime` is the ripple epoch time.
+- `PreviousTxnID` is the hash of the previous transaction to modify this entry. (Same as on other objects with this field.).
+- `PreviousTxnLgrSeq` is the ledger index of the ledger when this object was most recently updated/created. (Same as other objects with this field.)
+
+##### The `PriceOracle` Object ID Format
+
+We compute the `PriceOracle` object ID, a.k.a., `OracleID`, as the SHA-512Half of the following values, concatenated in order:
+
+- The Oracle space key (0x52)
+- The Owner Account ID
+- The Oracle Sequence. This field must be passed to the transactions and it describes a unique Price Oracle sequence for the given account.
+
+#### Example of `PriceOracle` JSON
+
+    {
+        "LedgerEntryType": "PriceOracle",
+        "OracleID": "00070C4495F14B0E44F78A264E41713C64B5F89242540EE255534400000000000000",
+        "Owner": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+        "Provider": "70726F7669646572",
+        "SymbolClass": "63757272656E6379",
+        "PriceDataSeries": [
+          {
+            "PriceData": {
+              "Symbol": "XRP",
+              "PriceUnit": "USD",
+              "SymbolPrice": 74,
+              "Scale": 2,
+            }
+          },
+        ],
+        "LastUpdateTime": 743609414,
+        "PreviousTxnID": "C53ECF838647FA5A4C780377025FEC7999AB4182590510CA461444B207AB74A9",
+        "PreviousTxnLgrSeq": 56865244
+    }
+
+## Transactions
+
+This proposal introduces several new transactions to allow for the creation, update, and deletion of the `PriceOracle` object.
+
+### Transaction for creating or updating `PriceOracle` instance
+
+We define a new transaction **SetOracle** for creating or updating a `PriceOracle` instance.  Before the transaction can be submitted to create a new `PriceOracle` instance, the Oracle Provider has to do the following:
+
+- Create or own the `Account` XRPL account with sufficient XRP balance to meet the XRP reserve and the transaction fee requirements.
+- The Oracle Provider has to publish the `Account` account public key so that it can be used for verification by dApp’s.
+- The Oracle Provider has to publish a registry of available Price Oracles with their unique Oracle Sequence. The hash of the `Account` and the `OracleSequence` uniquely identifies the Price Oracle on-ledger object.
+
+#### Transaction fields for **SetOracle** transaction
+
+|FieldName | Required? | JSON Type | Internal Type |
+|:---------|:-----------|:---------------|:------------|
+| `TransactionType` | :heavy_check_mark: | `string`| UINT16 |
+
+Indicates a new transaction type `SetOracle`. The integer value is 49.
```

Comment:
The transaction numbers need to be updated after DID. I recommend leaving them off the spec.